app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -254,12 +225,6 @@ def empty_file?(file)
         (file.respond_to?(:tempfile) && file.tempfile.size == 0) || (file.respond_to?(:size) && file.size == 0)
       end
 
-      # this is provided so that implementing application can override this behavior and map params to different attributes
```

Comment:
Do we think that the need expressed in this comment is no longer around? This ability to override the controller has been around for a long time, since https://github.com/samvera/sufia/commit/92a3788ec799183e862e1a636cdabebf305e5374.